style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+.navBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 5rem;
+}
+
+h1 {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  margin: 5rem 2rem 1rem 1rem;
+  width: 90%;
+}
+
+p {
+  color: #344563;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  margin: 1rem 2rem 1rem 1rem;
+  width: 75%;
+}
+
+h3 {
+  color: #7f8cff;
+  font-size: 1rem;
+  font-weight: 500;
+  margin-left: 1rem;
+}
+
+#iconListContainer {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.iconList {
+  margin-right: 2.5rem;
+  margin-left: -1.4rem;
+}
+
+.mainSection {
+  background-image: url(./img/background/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  background-size: 100vw;
+  width: 100vw;
+  height: 100vh;
+}
```

Comment:
- [ ] Please ensure that your background image matches the design layout, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Accepted type</td>
    </tr>
    <tr>
      <td>

 
![Student Design2](https://github.com/DamianDib9/personal-portfolio/assets/43528594/0c4c878c-3333-41d5-98ea-0486142420fb)
 
</td>
      <td>

![Figma Design](https://github.com/DamianDib9/personal-portfolio/assets/43528594/a0d1dd79-63dc-4bae-878f-fb158853ff9c)

</td>
    </tr>
  </table>